### PR TITLE
translated strings

### PR DESCRIPTION
**Pull request by mezysinc**
in ptbr

### CODE REVIEW

**Comment by fi3te** [comment]
Hi mezysinc,

thank you very much for your translations!

Best regards,
Fiete

File: app/src/main/res/values-pt-rBR/strings.xml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    <string name="notifications">Notificações</string>
+    <string name="cancel_automatically">Cancelar automaticamente</string>
+    <string name="display_duration_in_seconds">Exibir a duração em segundos</string>
+    <string name="invalid_duration">Duração inválida</string>
+    <string name="licenses">Licenças</string>
+    <string name="imprint">Sobre</string>
+    <string name="contact">E-mail: fiete.wennier@gmail.com \nTel.: sob pedido</string>
+    <string name="apache_license_hint">Sob a licença de Apache versão 2.0 (cópia abaixo)</string>
+    <string name="help">Ajuda</string>
+    - <string name="format_information">Formato: &lt;Segundos&gt; &lt;Minutos&gt; &lt;Horas&gt; &lt;Dia do mês&gt; &lt;Mês&gt; &lt;Dia da semana&gt; &lt;Ano (opcional)&gt;</string>
```
Comment (fi3te) [inline app/src/main/res/values-pt-rBR/strings.xml:34]: Please remove the unnecessary hyphen.

**Review by fi3te — CHANGES_REQUESTED** [review]

File: app/src/main/res/values-pt-rBR/strings.xml (hunk 1 above)
Comment (mezysinc) [reply app/src/main/res/values-pt-rBR/strings.xml:34]: Ok done.

Happy New Year : - )

Comment (mezysinc) [reply app/src/main/res/values-pt-rBR/strings.xml]: I guess there were 2 hyphens :)

**Review by fi3te — APPROVED** [review]

**Pull request merged**